src/components/textarea/TextArea.js

```diff
@@ -99,8 +107,19 @@ export class TextAreaComponent extends TextFieldComponent {
       return super.getValue();
     }
 
-    if (this.quill) {
-      return this.quill.root.innerHTML;
+    // FOR-998 Assigns default value
+    if (!this.data[this.component.key]) {
+      this.data[this.component.key] = "<p></p>";
```

Comment:
We want this for any non-wysiwyg TextArea's?.... I think this would set the default value of ```<p></p>``` inside of the normal TextArea won't it?